### ISSUE TITLE
Light Mode: TabItem selected

### ISSUE THREAD

**Issue by jasmeet21** [lepoco/wpfui#238]
**Describe the bug**  
Light mode missing indication of which TabItem is selected

**Expected behavior**  
On TabItem selection it should present darker color to represent which tab is selected

**Screenshots**  
Light Mode:
![image](https://user-images.githubusercontent.com/9377470/174521887-688ebb03-4fd6-4d54-ae6e-46ca5ccde2de.png)

Dark Mode:
![image](https://user-images.githubusercontent.com/9377470/174522039-c606b359-5871-4a1a-9edb-480f96636d28.png)

**Desktop (please complete the following information):**  
 - OS: [e.g. Windows 10]
 - .NET: [e.g. net5.0-windows10.0.17763.0]
 - Version: v1.x & v2.0.1

**Comment by MajorXAML** [comment]
Nope, the colors match the winui 3 style. But I would suggest removing the extra border at the bottom of the tabs.

![image_2022-06-20_105655651](https://user-images.githubusercontent.com/93913816/174553061-92d5d0fb-83c1-4c8a-bdcd-8a7a2f9f443f.png)

**Comment by jasmeet21** [comment]
I have not closely look into winui 3 colors but i think your suggestion would be far better instead of changing color as i suggested. Thanks MajorXAML

**Comment by ekdahl** [comment]
Used a color picker to compare the two light examples.

The WPF UI screenshot:
Selected tab color = FEFEFE (254)
Not selected tab color = FBFBFB (251)
Only 3 steps difference in the greyscale (I can hardly tell a difference on my monitor).

The WIN UI screenshot:
Selected tab color = F9F9F9 (249)
Not selected tab color = FBFBFB (243)
6 steps difference in the greyscale.

The difference between selected and not selected tab is twice as big in the WIN UI and much easier to see (also thanks to the missing line as you say).

**Comment by chucker** [comment]
2.0.1, Light:

<img width="830" height="492" alt="Image" src="https://github.com/user-attachments/assets/f6be9bb1-e53c-4236-a9d2-79e3da9f6849" />

2.0.1, Dark:

<img width="825" height="475" alt="Image" src="https://github.com/user-attachments/assets/e973e2ab-1794-4679-b614-caee708d8359" />

3.1.1, Light:

<img width="838" height="491" alt="Image" src="https://github.com/user-attachments/assets/77428114-9e09-49d3-be2a-ff0930de9cfa" />

3.1.1, Dark:

<img width="837" height="485" alt="Image" src="https://github.com/user-attachments/assets/e4f65ab8-0d4d-4044-af5c-08f8f517bac2" />

4.2.0, Light:

<img width="838" height="481" alt="Image" src="https://github.com/user-attachments/assets/811036b0-c117-4287-81c1-a4d862c59f69" />

4.2.0, Dark:

<img width="830" height="484" alt="Image" src="https://github.com/user-attachments/assets/78180575-3419-45ce-8cfd-dc672cf39e93" />

This is all with TabControl.

There's also TabView, but that doesn't seem to implement a style yet. And IMHO, TabView is a different thing. A TabView is for when you have multiple unrelated documents, with close buttons, etc., like in a web browser. The TabControl style should probably resemble that of WinUI 3's SelectorBar:

<img width="873" height="318" alt="Image" src="https://github.com/user-attachments/assets/58f15b43-9624-455b-82fd-0ef906628481" />
